server/api_router.go

```diff
@@ -93,6 +93,10 @@ func (p *Plugin) newAPIRouter() *mux.Router {
 	router.HandleFunc("/calls/{channel_id:[a-z0-9]{26}}/dismiss-notification", p.handleDismissNotification).Methods("POST")
 	router.HandleFunc("/calls/{call_id:[a-z0-9]{26}}/recording/{action}", p.handleRecordingAction).Methods("POST")
 
+	// Host Controls
+	hostCtrlRouter := router.PathPrefix("/host").Subrouter()
+	hostCtrlRouter.HandleFunc("/{call_id:[a-z0-9]{26}}/make_host/{new_host_id:[a-z0-9]{26}}", p.handleMakeHost).Methods("POST")
```

Comment:
Given hosts only make sense in the context of calls I think this endpoint should go under the `/calls` root.